README.tfdoc.hcl

```diff
@@ -95,6 +95,23 @@ section {
           END
         }
 
+        variable "module_timeouts" {
+          type           = any
+          readme_type    = "object(resource_name)"
```

Comment:
```suggestion
          readme_type    = "map(timeouts)"
```

Reply:
this is wrong in the template as well in this case